### ISSUE TITLE
Error opening keyboard settings

### ISSUE THREAD

**Issue by Eichhof** [rkkr/simple-keyboard#180]
Hello

I have incorporated your keyboard into my application. It works great but when I want to open the keyboard settings in the system settings of my phone, I'm getting the Toast message that the settings cannot be opened.

Which code is called when opening the keyboard settings via the system settings? It is not SettingsActivity.java (this activity is only called when opening the application).

Basically I have just copied all resources files from your app into the resource folder of my app and all the classes in a package of my app. I also have added everything to the manifest, except android.intent.action.MAIN and android.intent.category.LAUNCHER because that is already defined for my app.

**Comment by rkkr** [comment]
method.xml
